src/app/auth/components/singin/singin.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component } from '@angular/core';
+import { AuthService } from '../../services/auth.service';
+import { SignInBody } from '../../../core/types/auth.types';
+import { FormGroup, FormControl, Validators } from '@angular/forms';
+@Component({
+  selector: 'app-singin',
+  templateUrl: './singin.component.html',
+  styleUrls: ['./singin.component.scss'],
+})
+export class SinginComponent {
+  constructor(private auth: AuthService) {}
```

Comment:
Please try to follow a single approach to the declaration sequence. I can propose you the following

@decorated properties
public properties
private properties
constructor
lifecycle hooks
public methods
private methods